backbone/src/main/javaorg/researchstack/backbone/DataProvider.java

```diff
@@ -169,6 +206,13 @@ public static void init(DataProvider instance)
     public abstract User getUser(Context context);
 
     /**
+     * Saves the user object
+     * @param context android context
+     * @param user User object to save
+     */
+    public abstract void setUser(Context context, User user);
```

Comment:
I thought it was odd that this wasn't an abstract method, but I I think it was because all User object was before was name, email, birthday, which were set automatically by other methods in DataProvider.